src/status_im2/contexts/chat/messages/content/text/style.cljs

```diff
@@ -7,7 +7,10 @@
   {:margin-top -4})
 
 (def paragraph
-  {:margin-top spacing-between-blocks})
+  {:margin-top           spacing-between-blocks
+   :position             :relative
+   :text-align-vertical  :top
```

Comment:
@rahulpsingh, just curious, you probably tried `:text-align-vertical :center`. Does it lead to worse or better results?

From the docs:

>Set to false to remove extra font padding intended to make space for certain ascenders / descenders. With some fonts, this padding can make text look slightly misaligned when centered vertically. For best results also set textAlignVertical to center.

Reply:
It's not making any impact, so I removed it at all. The idea here was to remove the font padding from rest of the paragraph and make it centered so that it looks aligned in the Android devices.